Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1765,6 +1765,15 @@ - (void)_storeAppHighlight:(const WebCore::AppHighlight&)highlight
 }
 #endif
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+- (void)_removeStyleForID:(NSUUID *)uuid
```

Comment:
`_removeTextIndicatorStyleForID`? `Style` could mean anything. Same comment for similar method names.